index.html

```diff
@@ -28,7 +28,9 @@ <h2 id="presented_headline"> </h2>
         <input type="text" id="guess_box" name="guess"><br><br>
         <p id= "cross"></p>
         </div>
-        <button type="button" id="submit_button" onclick="onSubmit()"></button> 
```

Comment:
You've notw removed the submit button completely

Reply:
remind me how we move forward with this ticket? I now have a new issue with the 'submit with enter' feature. But I need to know how I resolve this before I can carry on